### PR TITLE
Alquileres

### PR DESCRIPTION
**Pull request by lucasmelul**
## Summary
View List Detail. Alquileres and Wishlist.
## Screenshots
imageedit_9_2082578726
imageedit_7_4076591201
## Trello Card
https://trello.com/c/QO7Zuvam/30-alquileres-y-wishlist

### CODE REVIEW

**Comment by lucasmelul** [comment]
I did the rebase

File: frontend-bootstrap/src/scss/base.scss [hunk 1]
```diff
@@ -82,6 +82,7 @@ body {
   font-size: 20px;
   font-weight: bold;
   height: 40px;
+  padding: 0px 25px;
```
Comment (SKOLZ) [inline frontend-bootstrap/src/scss/base.scss]: remove the px for 0 values

File: frontend-bootstrap/src/app/components/menu/bookDetail/bookDetail.scss [hunk 2]
```diff
@@ -53,24 +53,38 @@
...
+        letter-spacing: 0;
+        line-height: 19px;
+        padding: 10px 0;
+      }
+
+      .book-button-wish {
+        background: $color-button-wish;
+      }
+
+      .book-button-disabled {
```
Comment (SKOLZ) [inline frontend-bootstrap/src/app/components/menu/bookDetail/bookDetail.scss]: there's a pseudo class `:disabled` that applies the styles when the element is disabled

Comment (SKOLZ) [inline frontend-bootstrap/src/app/components/menu/bookDetail/bookDetail.scss]: you can remove the ng-class in the html and use that

File: frontend-bootstrap/src/app/components/menu/bookDetail/bookDetailController.js [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
-  function($http, $stateParams, bookService) {
+  '$http', '$stateParams', 'bookService', 'userService', '$rootScope',
+  function($http, $stateParams, bookService, userService, $rootScope) {
     this.bookId = $stateParams.bookId;
-
+    this.disabledBookBtn = false;
     bookService.getBookById(this.bookId).then((res) => {
       this.book = res.data;
+      if (this.book.actual_rent && this.book.actual_rent.user_id === $rootScope.userSession.id) {
+        this.disabledBookBtn = true;
```
Comment (SKOLZ) [inline frontend-bootstrap/src/app/components/menu/bookDetail/bookDetailController.js]: you can say:
```
this.disabledBookBtn = this.book.actual_rent && this.book.actual_rent.user_id === $rootScope.userSession.id;
```
and avoid the if

**Pull request merged**